app/web/components/EditLocationMap.test.tsx

```diff
@@ -11,7 +11,7 @@ import EditLocationMap from "./EditLocationMap";
 
 const { t } = i18n;
 
-jest.mock("components/Map");
```

Comment:
Just to know, will oldMap and Map coexist together?

Reply:
Temporarily! After this I will do more PRs to remove the old Map. I just wanted to focus only on the new map in this PR as it's already a lot of code.